R/reportPrices.R

```diff
@@ -672,13 +675,20 @@ reportPrices <- function(gdx, output=NULL, regionSubsetList=NULL,
   ## apply lowpass filter to receive moving average prices ----
   out.lowpass <- lowpass(out)
   ## add "Moving Avg" to variable name
-  getNames(out.lowpass) <- paste0(substr(getNames(out.lowpass),
-                                         1,nchar(getNames(out.lowpass))-13),
-                                  "|Moving Avg",
-                                  substr(getNames(out.lowpass),
-                                         nchar(getNames(out.lowpass))-12,
-                                         nchar(getNames(out.lowpass))))
+  getNames(out.lowpass) <- paste0(magclass::unitsplit(getNames(out))$variable,
+                                  "|Moving Avg (",
+                                  magclass::unitsplit(getNames(out))$unit, ")")
```

Comment:
The double colon operator is superfluous, as `magclass` is imported whole.  If anything, use `@importFrom`.

Reply:
Just a heads-up: magclass::unitsplit is the recommended way to do it (not importFrom). More details in the next RSE newsletter.

Reply:
Well, I'm excited for RSE's cogent argument for breaking R namespace conventions.